openwisp_controller/config/static/config/js/utils.js

```diff
@@ -53,10 +54,27 @@ evaluateVars = function (data, context) {
     return data;
 };
 
+getAllContext = function () {
+    var userContextField = getContext(),
+        systemContext = JSON.parse(document.getElementById('system_context').value),
+        value;
+    if (userContextField) {
+        var defaultValues = JSON.parse(userContextField.value);
+        value = Object.assign(
+            {},
+            systemContext,
+            defaultValues
+        );
+        return value;
+    } else {
+        return systemContext;
+    }
```

Comment:
here I would skip the last else since it's not needed and by avoiding it we can shrink the code a bit